<launch>
  <node pkg="localization" type="particle_filter.py" name="particle_filter" output="screen">
    <rosparam command="load" file="$(find localization)/params.yaml"/>
    <param name="particle_filter_frame" value="/hood_lidar"/>
    <param name="map_topic" value="/map"/>
    <param name="lidar_scale_to_map_scale" value="5.0"/>
    <param name="scan_topic" value="/tesse/hood_lidar/scan"/>
    <param name="odom_topic" value="/tesse/odom"/>
    <param name="angle_step" value="11"/>
    <param name="num_beams_per_particle" value="99"/>
  </node>
</launch>
